src/server/scripts/World/npcs_special.cpp

```diff
@@ -2192,7 +2192,7 @@ class npc_firework : public CreatureScript
                 // Check if we are near Elune'ara lake south, if so try to summon Omen or a minion
                 if (me->GetZoneId() == ZONE_MOONGLADE)
                 {
-                    if (!me->FindNearestCreature(NPC_OMEN, 100.0f, false) && me->GetDistance2d(omenSummonPos.GetPositionX(), omenSummonPos.GetPositionY()) <= 100.0f)
+                    if (!me->FindNearestCreature(NPC_OMEN, 100.0f, true) && me->GetDistance2d(omenSummonPos.GetPositionX(), omenSummonPos.GetPositionY()) <= 100.0f)
```

Comment:
Sorry for the inexperience but with the parameter of alive = false, being possible to create multiple Omen, even if there is already another (dead or alive). 
And it does not seem right, check if there is a dead Omen, after all is a world event and after a group kill Omen, other groups should be able to create others to do the same.